api/graphql/mutations/user.js

```diff
@@ -98,19 +79,23 @@ export const login = (fetchOne, { req }) => async (_, { email, password }) => {
   try {
     const isLoggedIn = await UserSession.isLoggedIn(req)
 
+    // Based upon current front-end implemenations this should never run,
+    // wondering if it might be better to logout and authenticate with the
```

Comment:
hmm, makes more sense to leave as is for me and make user explicitly tell us what they want to do, even though this should never happen right now

Reply:
First off, nothing to do with it for now anyway... 

But to the conversation: as long as the error got displayed in whatever theoretical bad front-end state this would happen in I guess this makes sense. I guess it would in our current Login forms. My counter argument is that if the user was sending us a valid email and password combination, presumably from our own form, logging them in would be the expected thing and what they were explicitly trying to do 🤷🏼‍♂️💨